search_engine_parser/core/utils.py

```diff
@@ -21,7 +22,13 @@
 
 
 def get_rand_user_agent():
-    return random.choice(USER_AGENT_LIST)
+    user_agent = random.choice(USER_AGENT_LIST)
+    try:
+        user_agent = UserAgent().random
+    except:
+       pass
```

Comment:
Does the library throw any exceptions? We could do 
```python
  user_agent = UserAgent().random
  If not user_agent:
    user_agent = random.choice(USER_AGENT_LIST)
```

Reply:
??

Reply:
Yeah it throws exceptions... hence the try except